unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/ECSRuntime/ComponentsManager/ECSComponentsManager.cs

```diff
@@ -130,5 +149,20 @@ public bool HasAnyComponent(IParcelScene scene, IDCLEntity entity)
             }
             return false;
         }
+
+        /// <summary>
+        /// creates a components group
+        /// </summary>
+        /// <param name="componentId1">id of one of the components</param>
+        /// <param name="componentId2">id of the other component</param>
+        /// <typeparam name="T1">model type of `componentId1`</typeparam>
+        /// <typeparam name="T2">model type of `componentId2`</typeparam>
+        /// <returns></returns>
+        public IECSReadOnlyComponentsGroupOfTwo<T1, T2> CreateComponentGroup<T1, T2>(int componentId1, int componentId2)
```

Comment:
I dont see this used outside the Test. Is this PR just scaffolding and preparing the future or did you forgot something?

Reply:
to keep the PR as small as I could I didn't introduce any change in the code to implement it.
so is more like "preparing for the near future, probably this week" :P